services/api/routes/public.js

```diff
@@ -26,6 +26,7 @@ var routes = [
     config: {
       validate: {
         payload: {
+          id: numericSchema.required(),
```

Comment:
postgreSQL `bigints` can get to be larger than the JS number type supports, so the numericSchema covers our butts should we ever encounter that problem by using:

```
Joi.alternatives().try(
  Joi.number().integer(),
  Joi.string().regex(/^\d+$/)
);
```

I can add in a positive rule as well.